cmd/policy/policy.go

```diff
@@ -134,9 +135,28 @@ func NewCommand(config *settings.Config, preRunE validator) *cobra.Command {
 		return cmd
 	}()
 
+	delete := func() *cobra.Command {
+		cmd := &cobra.Command{
+			Short: "Delete a policy",
+			Use:   "delete <policyID>",
+			RunE: func(cmd *cobra.Command, args []string) error {
+				err := policy.NewClient(*policyBaseURL, config).DeletePolicy(*ownerID, args[0])
+				if err != nil {
+					return fmt.Errorf("failed to delete policy: %v", err)
+				}
+				io.WriteString(cmd.OutOrStdout(), "Deleted Successfully\n")
```

Comment:
Let's send this to stderr. Logs or human readable messages should be in stderr and pipe-able structured data should be in stdout. Unix philosophy wise